R/meas-lcdm-crum.R

```diff
@@ -13,7 +13,7 @@
 #'   and `priors`.
 #' @rdname lcdm-crum
 #' @noRd
-meas_lcdm <- function(qmatrix, max_interaction = Inf, priors) {
+meas_lcdm <- function(qmatrix, max_interaction = Inf, priors, ncrum = FALSE) {
```

Comment:
Can we make this more generic, or explicit as to what this argument is doing? Something like `positive_interations`